### ISSUE TITLE
Update annnotations on gist:hasAccuracy

### ISSUE THREAD

**Issue by rjyounes** [semanticarts/gist#1146]
The `skos:definition` should not be limited to magnitudes. A measuring instrument or a measurement event could also have an accuracy.

Current:

```
gist:hasAccuracy
	a
		owl:ObjectProperty ,
		owl:FunctionalProperty
		;
	rdfs:range gist:Magnitude ;
	skos:definition "Relates a magnitude to the accuracy of its numeric value."^^xsd:string ;
	skos:example "Temperature accurate to tenth of a degree C; length accurate to the nearest centimeter."^^xsd:string ;
	skos:prefLabel "has accuracy"^^xsd:string ;
	skos:scopeNote """A typical way to use accuracy is to have it represent 2 standard deviations of the distribution of measurement errors. With this convention, when the measurement method is well-calibrated (has an average error of zero) and its errors have a normal distribution, there is a 95% chance that the actual error in measurement, in either direction, is less than the accuracy.

Note that the unit of measure of the accuracy has to be compatible with the unit of measure of the original magnitude (e.g. something measured in meters could have a accuracy in terms of millimeters or any other unit that measures distance)."""^^xsd:string ;
	.
```

Proposed:

```
gist:hasAccuracy
	a
		owl:ObjectProperty ,
		owl:FunctionalProperty
		;
	rdfs:range gist:Magnitude ;
        gist:domainIncludes gist:Magnitude ;
	skos:definition "Relates a resource such as a magnitude to the accuracy of its numeric value."^^xsd:string ;
	skos:example "Temperature accurate to tenth of a degree C; length accurate to the nearest centimeter."^^xsd:string ;
	skos:prefLabel "has accuracy"^^xsd:string ;
        skos:scopeNote "Could also apply to measurement events (activities) or measuring instruments." ;
	skos:scopeNote """A typical way to use accuracy is to have it represent 2 standard deviations of the distribution of measurement errors. With this convention, when the measurement method is well-calibrated (has an average error of zero) and its errors have a normal distribution, there is a 95% chance that the actual error in measurement, in either direction, is less than the accuracy.

Note that the unit of measure of the accuracy has to be compatible with the unit of measure of the original magnitude (e.g. something measured in meters could have a accuracy in terms of millimeters or any other unit that measures distance)."""^^xsd:string ;
	.
```

**Comment by MichaelSullivanArchitect** [comment]
[like]  Michael J. Sullivan reacted to your message:
________________________________
From: Rebecca Younes ***@***.***>
Sent: Tuesday, August 13, 2024 8:00:26 PM
To: semanticarts/gist ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [External] : [semanticarts/gist] Update annnotations on gist:hasAccuracy (Issue #1146)


The skos:definition should not be limited to magnitudes. A measuring instrument or a measurement event could also have an accuracy.

Current:

gist:hasAccuracy
        a
                owl:ObjectProperty ,
                owl:FunctionalProperty
                ;
        rdfs:range gist:Magnitude ;
        skos:definition "Relates a magnitude to the accuracy of its numeric value."^^xsd:string ;
        skos:example "Temperature accurate to tenth of a degree C; length accurate to the nearest centimeter."^^xsd:string ;
        skos:prefLabel "has accuracy"^^xsd:string ;
        skos:scopeNote """A typical way to use accuracy is to have it represent 2 standard deviations of the distribution of measurement errors. With this convention, when the measurement method is well-calibrated (has an average error of zero) and its errors have a normal distribution, there is a 95% chance that the actual error in measurement, in either direction, is less than the accuracy.

Note that the unit of measure of the accuracy has to be compatible with the unit of measure of the original magnitude (e.g. something measured in meters could have a accuracy in terms of millimeters or any other unit that measures distance)."""^^xsd:string ;
        .


Proposed:

gist:hasAccuracy
        a
                owl:ObjectProperty ,
                owl:FunctionalProperty
                ;
        rdfs:range gist:Magnitude ;
        gist:domainIncludes gist:Magnitude ;
        skos:definition "Relates a resource such as a magnitude to the accuracy of its numeric value."^^xsd:string ;
        skos:example "Temperature accurate to tenth of a degree C; length accurate to the nearest centimeter."^^xsd:string ;
        skos:prefLabel "has accuracy"^^xsd:string ;
        skos:scopeNote "Could also apply to measurement events (activities) or measuring instruments." ;
        skos:scopeNote """A typical way to use accuracy is to have it represent 2 standard deviations of the distribution of measurement errors. With this convention, when the measurement method is well-calibrated (has an average error of zero) and its errors have a normal distribution, there is a 95% chance that the actual error in measurement, in either direction, is less than the accuracy.

Note that the unit of measure of the accuracy has to be compatible with the unit of measure of the original magnitude (e.g. something measured in meters could have a accuracy in terms of millimeters or any other unit that measures distance)."""^^xsd:string ;
        .

**Comment by rjyounes** [comment]
Fix with #1149.

**Comment by MichaelSullivanArchitect** [comment]
Is there an example of an instance of an "accuracy" magnitude in the style of "plus or minus 5%" ?

**Comment by uscholdm** [comment]
> Is there an example of an instance of an "accuracy" magnitude in the style of "plus or minus 5%" ?

No, but that is an interesting idea. We do have something similar in the [operators ontology](https://github.com/semanticarts/operators-ontology). See the properties: 

- `hasTargetValue`
- `hasPercentVariation`
- `hasAbsoluteVariation`

**Comment by MichaelSullivanArchitect** [comment]
Regarding "This is an abstract superproperty, not intended for use in triples" I suppose this means that we should create our own subproperty based on this – correct? And in the use-case of a precision of "plus or minus 5%" what exactly would that subproperty reference (i.e. its rdfs:range)? Some sort of Magnitude? Blank node? Or something else?

M


________________________________
From: Michael Uschold ***@***.***>
Sent: Wednesday, September 11, 2024 3:25 PM
To: semanticarts/gist ***@***.***>
Cc: Michael J. Sullivan ***@***.***>; Comment ***@***.***>
Subject: [External] : Re: [semanticarts/gist] Update annnotations on gist:hasAccuracy (Issue #1146)


Is there an example of an instance of an "accuracy" magnitude in the style of "plus or minus 5%" ?

No, but that is an interesting idea. We do have something similar in the operators ontology<https://urldefense.com/v3/__https://github.com/semanticarts/operators-ontology__;!!ACWV5N9M2RV99hQ!I7OtTR8UEY0jAfKogMvJH8dhOblmMymd_VVCs20gWzqSy7nDqsvTlRUSfLoUVj14m74tpGTfQir50jjEXLHLChcvD0dDzhU$>. See the properties:

  *   hasTargetValue
  *   hasPercentVariation
  *   hasAbsoluteVariation

**Comment by uscholdm** [comment]
The second `scopeNote` is far more technical and detailed than is the norm for gist.  I read it several times, I have a math background, and I still can't work out exactly what it means or how to make use of it. If this information is likely to be useful to gist users, I propose to use `rdf:seeAlso` to a link that has a good explanation. In the meantime, here is my proposal:

```
gist:hasAccuracy
	a
		owl:ObjectProperty ,
		owl:FunctionalProperty
		;
	rdfs:range gist:Magnitude ;
        gist:domainIncludes gist:Magnitude ;
	skos:definition "Relates a resource such as a magnitude to the accuracy of its numeric value."^^xsd:string ;
	skos:example "Temperature accurate to tenth of a degree C; length accurate to the nearest centimeter."^^xsd:string ;
	skos:prefLabel "has accuracy"^^xsd:string ;
        skos:scopeNote "Could also apply to measurement events (activities) or measuring instruments." ;
	skos:scopeNote "Note that the unit of measure of the accuracy has to be compatible with the unit of measure of the original magnitude (e.g. something measured in meters could have a accuracy in terms of millimeters or any other unit that measures distance)."^^xsd:string ;
	.
```

**Comment by rjyounes** [comment]
I agree that the original is too technical. Do you think "compatible" needs to be elaborated on or is it self-explanatory enough?

**Comment by philblackwood** [comment]
gist should use the ISO definition, and we can use the following as a seeAlso: 
[
](https://en.wikipedia.org/wiki/Accuracy_and_precision)

Definition: The accuracy of a measurement method or device is the degree of closeness of its measurements to the actual values of the aspects being measured. The term accuracy can also refer to the likely closeness of an individual measurement to the actual value. In the ISO standards, accuracy is also called trueness.

Scope note: the accuracy is relative to a fixed number of standard deviations in the distribution of errors, typically chosen as 2 sigma. With this convention, about 95% of measurement errors will be less than the value of the accuracy.

**Comment by stevenchalem** [comment]
[like]  Steven Chalem reacted to your message:
________________________________

**Comment by uscholdm** [comment]
I am unfamiliar with accuracy being defined in terms of % and probability distributions. I always took it to mean accurate to so many digits - but I guess that is precision.  Precision seems useful and easy to understand and use and makes sense to have in gist. I'm less certain about the usefuness of having accuracy  in gist.

**Comment by MichaelSullivanArchitect** [comment]
I feel the concepts of adding Accuracy and Precision are an attempt to complete the UOM efforts so that it is not just an enumerated list but rather something much richer. OTOH, Accuracy can indeed be a complex subject.
